TestApp/Model/TestAPIContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TestApp.Model
+{
+    public class TestAPIContext : DbContext
+    {
+        public TestAPIContext(DbContextOptions options)
+            : base(options)
+        { }
+        public DbSet<Person> Persons { get; set; }
+        public DbSet<Identifier> Identifier { get; set; }
```

Comment:
change to identifiers

Reply:
changing to Identifier to Identifiers was giving error , Invalid Object.